roles/ansible/tower/config-ansible-tower-ocp/tasks/deploy_tower.yml

```diff
@@ -49,3 +49,7 @@
   args:
     chdir: "{{ ansible_tower_dir }}"
 
+- name: "Pausing for 5 minutes to let Tower settle down"
+  pause:
+    minutes: 5
```

Comment:
Would it make sense to allow users to configure this in the inventory by adding a variable and defaulting to 5 minutes or 300 seconds or similar?

Reply:
Sure, I will add that

Reply:
Done